src/graph.py

```diff
@@ -345,16 +350,20 @@ def output_forest_as_networkx(self, vertex_indices, edge_indices):
             networkx.Graph: a networkx graph object
         """
 
+        if len(vertex_indices) == 0: 
+            logger.info("The resulting Forest is empty. Try different parameters.")
```

Comment:
This help message might not be useful in the context of grid randomizations. Makes sense to remove it, imo.

Reply:
Sure: